lib/service/Service.ts

```diff
@@ -81,7 +81,7 @@ class Service {
     const { currency, swapClient, tokenAddress, decimalPlaces } = args;
 
     let address = tokenAddress;
-    if (args.swapClient === SwapClientType.Raiden && address) {
+    if (args.swapClient === SwapClientType.Connext && address) {
```

Comment:
```suggestion
    if ((args.swapClient === SwapClientType.Connext || args.swapClient === SwapClientType.Raiden) && address) {
```
Technically we still support Raiden

Reply:
I think we agreed to drop it all and other PRs have removed bits of support, so I'm fine with this as is and then cleaning up the remaining portions in a separate PR.

Reply:
Nevermind then